### PR TITLE
fix(custom-query) : Input gets cleared after saving custom-query

### PR DESCRIPTION
**Pull request by divyanshiGupta**
Fixes https://github.com/openshiftio/openshift.io/issues/3741

Input for custom-query name gets cleared once the query is saved and doesn't show up in the input field of the save custom-query drop-down the next time the save custom-query drop-down is opened.

### CODE REVIEW

**Comment by alien-ike** [comment]
### Ike Plugins (test-keeper)

Thank you @divyanshiGupta for this contribution!



It appears that no tests have been added or updated in this PR.

Automated tests give us confidence in shipping reliable software. Please add some as part of this change.

If you are an admin or the reviewer of this PR and you are sure that no test is needed then you can use the command `/ok-without-tests` as a comment to make the status green.


For more information please head over to official [documentation](http://arquillian.org/ike-prow-plugins/#_test_keeper_plugin). You can find there how to configure the plugin.

**Comment by Raunak1203** [comment]
@divyanshiGupta @nimishamukherjee should we also clear the input on Cancel?

**Comment by divyanshiGupta** [comment]
@Raunak1203 I implemented clear input on cancel as well but I noticed that you have mentioned only for save that is why I removed it. I think it should get cleared on cancel as well but let's wait for @nimishamukherjee to reply.

File: src/app/components_ngrx/toolbar-panel/toolbar-panel.component.html [hunk 1]
```diff
@@ -46,12 +46,12 @@
         </aside>
         <div>
           <input class="form-control ng-pristine ng-valid ng-touched"
-          (keyup.enter)="saveFilters(filterSaveInp.value)"
-          #filterSaveInp />
+          (keyup.enter)="saveFilters(filterSaveInp,filterSaveInp.value)"
```
Comment (sanbornsen) [inline src/app/components_ngrx/toolbar-panel/toolbar-panel.component.html]: Missing space after the `,`

Comment (sanbornsen) [reply src/app/components_ngrx/toolbar-panel/toolbar-panel.component.html]: If we are already passing `filterSaveInp` what is the need of passing a property of it separately?

File: src/app/components_ngrx/toolbar-panel/toolbar-panel.component.html [hunk 2]
```diff
@@ -46,12 +46,12 @@
...
+          #filterSaveInp/>
```
Comment (sanbornsen) [inline src/app/components_ngrx/toolbar-panel/toolbar-panel.component.html]: Why was the space removed?

File: src/app/components_ngrx/toolbar-panel/toolbar-panel.component.html [hunk 3]
```diff
@@ -46,12 +46,12 @@
...
         </div>
         <div class="save-filter-action">
           <button class="btn btn-primary pull-right margin-left-5 save-cq-btn"
-            (click)="saveFilters(filterSaveInp.value)"
+            (click)="saveFilters(filterSaveInp,filterSaveInp.value)"
```
Comment (sanbornsen) [inline src/app/components_ngrx/toolbar-panel/toolbar-panel.component.html]: Same as above comments

File: src/app/components_ngrx/toolbar-panel/toolbar-panel.component.ts [hunk 4]
```diff
@@ -602,7 +602,7 @@ export class ToolbarPanelComponent implements OnInit, AfterViewInit, OnDestroy {
     });
   }
 
-  saveFilters(filterName: string) {
+  saveFilters(filterSaveInp,filterName: string) {
```
Comment (sanbornsen) [inline src/app/components_ngrx/toolbar-panel/toolbar-panel.component.ts]: Where is the type?

Comment (sanbornsen) [inline src/app/components_ngrx/toolbar-panel/toolbar-panel.component.ts]: Space missing after the `,`

File: src/app/components_ngrx/toolbar-panel/toolbar-panel.component.ts [hunk 5]
```diff
@@ -617,6 +617,9 @@ export class ToolbarPanelComponent implements OnInit, AfterViewInit, OnDestroy {
       };
       this.store.dispatch(new CustomQueryActions.Add(customQuery));
       this.closeFilterSave();
+      if(filterSaveInp) {
```
Comment (sanbornsen) [inline src/app/components_ngrx/toolbar-panel/toolbar-panel.component.ts]: Why is the check needed?

**Comment by divyanshiGupta** [comment]
@sanbornsen made the changes.

**Comment by nimishamukherjee** [comment]
@divyanshiGupta yes please clear on cancel.

**Pull request merged**